MerlinAU.sh

```diff
@@ -3139,17 +3139,18 @@ _EstimateNextCronTimeAfterDate_()
     while [ "$found" = "false" ]
     do
         loopCount="$((loopCount + 1))"
+
         if matches_month "$current_month" "$month_field" && \
            matches_day_of_month "$current_day" "$dom_field" && \
            matches_day_of_week "$current_dow" "$dow_field"
         then
             for this_hour in $(expand_cron_field "$hour_field" 0 23)
             do
-                if [ "$this_hour" -gt "$current_hour" ]
+                if [ "$this_hour" -gt "$current_hour" ] || [ "$loopCount" -gt 1 ]
```

Comment:
Ah, OK. The check for "`$loopCount" -gt 1`" essentially bypasses the rest of the code, which is why it works.

Reply:
> Ah, OK. The check for "`$loopCount" -gt 1`" essentially bypasses the rest of the code, which is why it works.
>
The more I think about it, the more I come to realize that bypassing the rest of the code when the `loopCount` reaches 2 
is not quite the right fix. I think the fix is further down to checking the minutes.

I'll submit a PR with my proposed solution and see how it works for you.

Reply:
@Martinski4GitHub 

Received.

I'll let the master cook, I'll check back later! 😉